### PR TITLE
Explanatory text for 1.0 readme

### PR DESCRIPTION
**Pull request by probins**
Some more help for #378

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
 # ES6 Module Loader Polyfill [![Build Status][travis-image]][travis-url]
 
-_For upgrading to ES6 Module Loader 0.16, [read the release notes here](https://github.com/ModuleLoader/es6-module-loader/releases/tag/v0.16.0)._
```
Comment (guybedford) [inline README.md]: This line is needed.

File: README.md [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
-
 Dynamically loads ES6 modules in browsers and [NodeJS](#nodejs-use) with support for loading existing and custom module formats through loader hooks.
 
-This project implements dynamic module loading through `System` exactly to the previous ES6-specified loader API at [2014-08-24 ES6 Specification Draft Rev 27, Section 15](http://wiki.ecmascript.org/doku.php?id=harmony:specification_drafts#august_24_2014_draft_rev_27) and is being converted to track the newly redrafted specification at https://github.com/whatwg/loader (work in progress at https://github.com/ModuleLoader/es6-module-loader/pull/317).
+This project implements dynamic module loading as per the newly redrafted specification at [WhatWG loader spec](https://github.com/whatwg/loader). It replaces the 0.* branch, which implements the previous ES6-specified loader API at [2014-08-24 ES6 Specification Draft Rev 27, Section 15](http://wiki.ecmascript.org/doku.php?id=harmony:specification_drafts#august_24_2014_draft_rev_27).
+
+The rest of this Readme describes the previous implementation and will be updated as work progresses.
```
Comment (guybedford) [inline README.md]: How about `All documentation here relates to the prior specification API, which will upgrade to the latest specification as soon as it is ready.`?

File: README.md (hunk 1 above)
Comment (probins) [inline README.md]: why is this needed in the readme for version 1?

**Comment by guybedford** [comment]
Oh sorry, I misread that. Thanks!
